x-pack/platform/plugins/shared/fleet/server/saved_objects/index.ts

```diff
@@ -1114,6 +1114,29 @@ export const getSavedObjectTypes = (
           host_urls: { type: 'keyword', index: false },
           is_preconfigured: { type: 'boolean' },
           proxy_id: { type: 'keyword' },
+          ssl: { type: 'binary' },
```

Comment:
Should we use `dynamic:false` in this SO type too, like it's done for output? It makes it simpler to add new fields without listing them in mappings https://github.com/elastic/kibana/pull/208516#discussion_r1934356677

Reply:
That's a good one, I'll update it!

Reply:
it's not needed to add the `ssl` and `secrets` fields to mappings with `dynamic:false`, and can be removed from `current_fields.json` and `current_mappings.json` too